//===- ImageByteMap.proto ---------------------------------------*- C++ -*-===//
//
//  Copyright (C) 2018 GrammaTech, Inc.
//
//  This code is licensed under the MIT license. See the LICENSE file in the
//  project root for license terms.
//
//  This project is sponsored by the Office of Naval Research, One Liberty
//  Center, 875 N. Randolph Street, Arlington, VA 22203 under contract #
//  N68335-17-C-0700.  The content of the information does not necessarily
//  reflect the position or policy of the Government and no official
//  endorsement should be inferred.
//
//===----------------------------------------------------------------------===//
syntax = "proto3";
package protoV0;

import "ByteMap.proto";

message ImageByteMap
{
    bytes uuid = 1;
    ByteMap byte_map = 2;
    uint64 addr_min = 3;
    uint64 addr_max = 4;
    uint64 base_address = 5;
    uint64 entry_point_address = 6;
}
